frontend/src/Editor/AppVersionsManager.jsx

```diff
@@ -68,33 +68,44 @@ export const AppVersionsManager = function AppVersionsManager({
   };
 
   const createVersion = (versionName, createAppVersionFrom) => {
-    if (versionName.trim() !== '') {
-      setIsCreatingVersion(true);
-      appVersionService
-        .create(appId, versionName, createAppVersionFrom.id)
-        .then(() => {
-          closeModal();
-          toast.success('Version Created');
+    const trimmedVersionName = versionName.trim();
+    const isEmpty = !trimmedVersionName.length;
 
-          appVersionService.getAll(appId).then((data) => {
-            setAppVersions(data.versions);
+    if (isEmpty) {
+      toast.error('The name of version should not be empty');
+      return;
+    }
 
-            const latestVersion = data.versions.at(0);
-            setAppDefinitionFromVersion(latestVersion);
-            setEditingAppVersion(latestVersion);
-            setVersionName('');
-          });
+    const passCharLimit = trimmedVersionName.length > 255;
```

Comment:
It will be more readable if we can rename the variable to `passwordCharacterLimit`.

Reply:
Umm, this is not related to `password` but more of the app version name char limit. So maybe `charLimitReached`?

Reply:
oh sorry, I read it as "password character limit". `isWithinCharacterLimit` or something similar works?

Reply:
Yeah, this works.

Reply:
Done